### PR TITLE
[4.0] administrator/language/com_content

### PR DESCRIPTION
**Pull request by tecpromotion**
Pull Request für Issue #804 .

### CODE REVIEW

File: administrator/language/de-DE/com_content.ini [hunk 1]
```diff
@@ -5,21 +5,23 @@
...
 COM_CONTENT_ATTRIBS_FIELDSET_LABEL="Optionen"
-; COM_CONTENT_BATCH_MENU_LABEL is deprecated, use JLIB_HTML_BATCH_MENU_LABEL instead.
-COM_CONTENT_BATCH_MENU_LABEL="Zum Verschieben oder Kopieren der Auswahl eine Kategorie auswählen."
 COM_CONTENT_BATCH_OPTIONS="Mehrere ausgewählte Beiträge gleichzeitig bearbeiten (Stapelverarbeitung)"
 COM_CONTENT_BATCH_TIP="Wenn ein Beitrag kopiert/verschoben wird, so werden auch alle zusätzlich ausgewählten Aktionen auf den kopierten/verschobenen Beitrag angewendet. Ansonsten werden die Aktionen nur auf die ausgewählten Beiträge angewendet."
-COM_CONTENT_CHANGE_ARTICLE="Beitrag auswählen oder wechseln"
-COM_CONTENT_CHANGE_ARTICLE_BUTTON="Auswählen / Wechseln"
-COM_CONTENT_CHOOSE_CATEGORY_DESC="Übergeordnete Kategorie auswählen"
+COM_CONTENT_CHANGE_STAGE="Status wechseln"
+COM_CONTENT_CHANGE_STAGE_AMBIGUOUS_TRANSITIONS="Einige Übergänge sind für diesen Zustand zweideutig. Bitte den bevorzugten Übergang auswählen und fortfahren."
```
Comment (zero-24) [inline administrator/language/de-DE/com_content.ini:23]:
```suggestion
COM_CONTENT_CHANGE_STAGE_AMBIGUOUS_TRANSITIONS="Die Übergänge sind für diesen Zustand nicht eindeutig. Bitte den bevorzugten Übergang auswählen und fortfahren."
```

File: administrator/language/de-DE/com_content.ini [hunk 2]
```diff
@@ -28,187 +30,169 @@ COM_CONTENT_CONFIG_CATEGORY_SETTINGS_DESC="Diese Einstellungen gelten für eine
...
 COM_CONTENT_EDIT_CATEGORY="Kategorie bearbeiten"
-COM_CONTENT_EDITORCONFIG_FIELDSET_LABEL="Bearbeitungsanzeige konfigurieren"
 COM_CONTENT_EDITING_LAYOUT="Bearbeitungslayout"
-COM_CONTENT_ERROR_ALL_LANGUAGE_ASSOCIATED="Ein Inhalt der für „Alle“ Sprachen definiert wurde, kann <strong><u>nicht</u></strong> verknüpft werden. Die Verknüpfungen wurden <strong><u>nicht</u></strong> übernommen!"
+COM_CONTENT_EDITORCONFIG_FIELDSET_LABEL="Bearbeitungsanzeige konfigurieren"
+COM_CONTENT_EDIT_ARTICLE="Artikel bearbeiten"
+COM_CONTENT_ERROR_ALL_LANGUAGE_ASSOCIATED="Ein Inhalt der für „Alle“ Sprachen definiert wurde, kann <strong><u>nicht</u></strong> verknüpft werden. Die Verknüpfungen wurden <strong><u>nicht</u></strong> übernommen."
+COM_CONTENT_ERROR_CANNOT_ARCHIVE="Einer oder mehrere der ausgewählten Beiträge können nicht auf archiviert gesetzt werden."
+COM_CONTENT_ERROR_CANNOT_PUBLISH="Einer oder mehrere der ausgewählten Beiträge können nicht auf veröffentlicht gesetzt werden."
+COM_CONTENT_ERROR_CANNOT_TRASH="Einer oder mehrere der ausgewählten Beiträge können nicht auf gelöscht gesetzt werden."
```
Comment (zero-24) [inline administrator/language/de-DE/com_content.ini:48]:
```suggestion
COM_CONTENT_ERROR_CANNOT_TRASH="Einer oder mehrere der ausgewählten Beiträge können gelöscht werden."
```

File: administrator/language/de-DE/com_content.ini [hunk 3]
```diff
@@ -28,187 +30,169 @@ COM_CONTENT_CONFIG_CATEGORY_SETTINGS_DESC="Diese Einstellungen gelten für eine
...
-COM_CONTENT_DRILL_DOWN_LAYOUT_LABEL="Listen- oder Blog-Layout"
-COM_CONTENT_EDIT_ARTICLE="Beitrag bearbeiten"
 COM_CONTENT_EDIT_CATEGORY="Kategorie bearbeiten"
-COM_CONTENT_EDITORCONFIG_FIELDSET_LABEL="Bearbeitungsanzeige konfigurieren"
 COM_CONTENT_EDITING_LAYOUT="Bearbeitungslayout"
-COM_CONTENT_ERROR_ALL_LANGUAGE_ASSOCIATED="Ein Inhalt der für „Alle“ Sprachen definiert wurde, kann <strong><u>nicht</u></strong> verknüpft werden. Die Verknüpfungen wurden <strong><u>nicht</u></strong> übernommen!"
+COM_CONTENT_EDITORCONFIG_FIELDSET_LABEL="Bearbeitungsanzeige konfigurieren"
+COM_CONTENT_EDIT_ARTICLE="Artikel bearbeiten"
+COM_CONTENT_ERROR_ALL_LANGUAGE_ASSOCIATED="Ein Inhalt der für „Alle“ Sprachen definiert wurde, kann <strong><u>nicht</u></strong> verknüpft werden. Die Verknüpfungen wurden <strong><u>nicht</u></strong> übernommen."
+COM_CONTENT_ERROR_CANNOT_ARCHIVE="Einer oder mehrere der ausgewählten Beiträge können nicht auf archiviert gesetzt werden."
```
Comment (zero-24) [inline administrator/language/de-DE/com_content.ini:46]:
```suggestion
COM_CONTENT_ERROR_CANNOT_ARCHIVE="Einer oder mehrere der ausgewählten Beiträge können nicht archiviert werden."
```

File: administrator/language/de-DE/com_content.ini [hunk 4]
```diff
@@ -28,187 +30,169 @@ COM_CONTENT_CONFIG_CATEGORY_SETTINGS_DESC="Diese Einstellungen gelten für eine
...
+COM_CONTENT_ERROR_CANNOT_PUBLISH="Einer oder mehrere der ausgewählten Beiträge können nicht auf veröffentlicht gesetzt werden."
```
Comment (zero-24) [inline administrator/language/de-DE/com_content.ini:47]:
```suggestion
COM_CONTENT_ERROR_CANNOT_PUBLISH="Einer oder mehrere der ausgewählten Beiträge können nicht veröffentlicht werden."
```

File: administrator/language/de-DE/com_content.ini [hunk 5]
```diff
@@ -28,187 +30,169 @@ COM_CONTENT_CONFIG_CATEGORY_SETTINGS_DESC="Diese Einstellungen gelten für eine
...
+COM_CONTENT_ERROR_CANNOT_TRASH="Einer oder mehrere der ausgewählten Beiträge können nicht auf gelöscht gesetzt werden."
+COM_CONTENT_ERROR_CANNOT_UNPUBLISH="Einer oder mehrere der ausgewählten Beiträge können nicht auf unveröffentlicht gesetzt werden."
```
Comment (zero-24) [inline administrator/language/de-DE/com_content.ini:49]:
```suggestion
COM_CONTENT_ERROR_CANNOT_UNPUBLISH="Einer oder mehrere der ausgewählten Beiträge können nicht versteckt werde."
```

File: administrator/language/de-DE/com_content.ini [hunk 6]
```diff
@@ -28,187 +30,169 @@ COM_CONTENT_CONFIG_CATEGORY_SETTINGS_DESC="Diese Einstellungen gelten für eine
...
+COM_CONTENT_ERROR_UPDATE_STATE="Es konnte keine Status für das Element gesetzt werden"
```
Comment (zero-24) [inline administrator/language/de-DE/com_content.ini:50]:
```suggestion
COM_CONTENT_ERROR_UPDATE_STATE="Der Status für das Element konnte nicht geändert werden"
```

Element?

File: administrator/language/de-DE/com_content.ini [hunk 7]
```diff
@@ -28,187 +30,169 @@ COM_CONTENT_CONFIG_CATEGORY_SETTINGS_DESC="Diese Einstellungen gelten für eine
...
-COM_CONTENT_URL_FIELD_BROWSERNAV_DESC="Bestimmt das Zielfenster, das im Browser nach dem Anklicken des Links geöffnet wird."
+COM_CONTENT_UNPUBLISHED="Versteckt"
 COM_CONTENT_URL_FIELD_A_BROWSERNAV_LABEL="URL A Zielfenster"
+COM_CONTENT_URL_FIELD_BROWSERNAV_LABEL="URL-Zielfenster"
 COM_CONTENT_URL_FIELD_B_BROWSERNAV_LABEL="URL B Zielfenster"
 COM_CONTENT_URL_FIELD_C_BROWSERNAV_LABEL="URL C Zielfenster"
 COM_CONTENT_WARNING_PROVIDE_VALID_NAME="Bitte geben Sie einen gültigen, nicht leeren Titel an."
+COM_CONTENT_WORKFLOW="Workflow"
+COM_CONTENT_WORKFLOW_NOT_FOUND="Kein Standard-Workflow verfügbar, bitte einen definieren oder an einen Administrator wenden."
+COM_CONTENT_WORKFLOW_TRANSITION_NOT_ALLOWED="Es <strong><u>fehlt</u></strong> die notwendige Berechtigung diesen Übergang durchzuführen"
```
Comment (zero-24) [inline administrator/language/de-DE/com_content.ini:183]:
```suggestion
COM_CONTENT_WORKFLOW_TRANSITION_NOT_ALLOWED="Es <strong><u>fehlt</u></strong> die notwendige Berechtigung diesen Übergang auszuführen"
```

File: administrator/language/de-DE/com_content.ini [hunk 8]
```diff
@@ -28,187 +30,169 @@ COM_CONTENT_CONFIG_CATEGORY_SETTINGS_DESC="Diese Einstellungen gelten für eine
...
+JLIB_APPLICATION_ERROR_BATCH_CANNOT_EDIT="Es <strong><u>fehlt</u></strong> die notwendige Berechtigung zum Bearbeiten einer oder mehrerer dieser Beiträge."
+JLIB_APPLICATION_ERROR_BATCH_CANNOT_EXECUTE_TRANSITION="Es <strong><u>fehlt</u></strong> die notwendige Berechtigung um den Übergang für einen oder mehrere dieser Beiträge durchzuführen."
```
Comment (zero-24) [inline administrator/language/de-DE/com_content.ini:189]:
```suggestion
JLIB_APPLICATION_ERROR_BATCH_CANNOT_EXECUTE_TRANSITION="Es <strong><u>fehlt</u></strong> die notwendige Berechtigung um den Übergang für einen oder mehrere dieser Beiträge auszuführen."
```

File: administrator/language/de-DE/com_content.ini (hunk 5 above)
Comment (tecpromotion) [reply administrator/language/de-DE/com_content.ini:49]:
```suggestion
COM_CONTENT_ERROR_CANNOT_UNPUBLISH="Einer oder mehrere der ausgewählten Beiträge können nicht versteckt werden."
```

File: administrator/language/de-DE/com_content.ini (hunk 2 above)
Comment (tecpromotion) [reply administrator/language/de-DE/com_content.ini:48]:
```suggestion
COM_CONTENT_ERROR_CANNOT_TRASH="Einer oder mehrere der ausgewählten Beiträge können nicht gelöscht werden."
```

**Comment by tecpromotion** [comment]
thanks @zero-24 for suggestions

**Comment by zero-24** [comment]
Merged :+1:

**Pull request merged**